sdk/metrics/src/jmhBasedTest/java/io/opentelemetry/sdk/metrics/internal/state/TestInstrumentType.java

```diff
@@ -32,11 +34,36 @@ InstrumentTester createInstrumentTester() {
     InstrumentTester createInstrumentTester() {
       return new ExplicitBucketHistogramTester();
     }
+  },
+  LONG_SUM(/* dataAllocRateReductionPercentage= */ 97.3f) {
+    @Override
+    InstrumentTester createInstrumentTester() {
+      return new LongSumTester();
+    }
+  },
+  DOUBLE_SUM(/* dataAllocRateReductionPercentage= */ 97.3f) {
+    @Override
+    InstrumentTester createInstrumentTester() {
+      return new DoubleSumTester();
+    }
   };
 
-  abstract InstrumentTester createInstrumentTester();
+  private final float dataAllocRateReductionPercentage;
+
+  TestInstrumentType() {
+    this.dataAllocRateReductionPercentage = 99.8f; // default
+  }
+
+  // Some instruments have different reduction percentage.
+  TestInstrumentType(float dataAllocRateReductionPercentage) {
+    this.dataAllocRateReductionPercentage = dataAllocRateReductionPercentage;
+  }
+
+  float getDataAllocRateReductionPercentage() {
+    return dataAllocRateReductionPercentage;
+  }
 
-  TestInstrumentType() {}
+  abstract InstrumentTester createInstrumentTester();
```

Comment:
Non-blocking comment: I know the structure of this class is following the error prone advice here: https://errorprone.info/bugpattern/ImmutableEnumChecker

But I'm not sure I buy that advice. Reads much cleaner if you ignore it:
```
@SuppressWarnings("ImmutableEnumChecker")
public enum TestInstrumentType {
  ASYNC_COUNTER(AsyncCounterTester::new),
  EXPONENTIAL_HISTOGRAM(ExponentialHistogramTester::new),
  EXPLICIT_BUCKET(ExplicitBucketHistogramTester::new),
  LONG_SUM(LongSumTester::new, /* dataAllocRateReductionPercentage= */ 97.3f),
  DOUBLE_SUM(DoubleSumTester::new, /* dataAllocRateReductionPercentage= */ 97.3f);

  private final Supplier<? extends InstrumentTester> testerInitializer;
  private final float dataAllocRateReductionPercentage;

  TestInstrumentType(Supplier<? extends InstrumentTester> testerInitializer) {
    this(testerInitializer, 99.8f);
  }

  // Some instruments have different reduction percentage.
  TestInstrumentType(
      Supplier<? extends InstrumentTester> testerInitializer,
      float dataAllocRateReductionPercentage) {
    this.testerInitializer = testerInitializer;
    this.dataAllocRateReductionPercentage = dataAllocRateReductionPercentage;
  }

  float getDataAllocRateReductionPercentage() {
    return dataAllocRateReductionPercentage;
  }

  InstrumentTester createInstrumentTester() {
    return testerInitializer.get();
  }
```

Reply:
You are correct. I fixed it.